src/main/java/lotto/domain/LottoStore.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import lotto.exception.LottoException;
+import lotto.util.LottoNumberGenerator;
+
+public class LottoStore {
+    private final int LOTTO_PRICE = 1_000;
+
+    public Lottos sell(int purchaseAmount) {
```

Comment:
보통 메서드는 외부에 사용할 수 있도록 "제공"하는 개념이라서 네이밍도 사용자 입장에서 작성하는 편입니다.
`LottoStore`가 제공해 주어야 하는 메서드는 "구매"를 할 수 있는 메서드이겠네요.
```suggestion
    public Lottos purchase(int purchaseAmount) {
```